slurm/README.gracc-fields

```diff
@@ -0,0 +1,78 @@
+Here is a sample raw slurm record from GRACC --
+
+https://gracc.opensciencegrid.org/kibana/goto/3849a063b7e79df2f559db6ba3c2a2dd
+
+
+Field Name                       Value
+----------                       -----
+CoreHours                        3.32
+CpuDuration                      63.92
+CpuDuration_system               29
+CpuDuration_system_description   Was entered in seconds
+CpuDuration_user                 34.92
+CpuDuration_user_description     Was entered in seconds
+CreateTime                       March 26th 2019, 19:43:54.000
+DN                               /DC=ch/DC=cern/OU=computers/CN=cmspilot01/vocms080.cern.ch
+EndTime                          March 26th 2019, 19:42:26.000
+GlobalJobId                      slurm:slurmdb-mgmt/3307/slurm/hipergator.34033351
+Grid                             OSG
+JobName                          bl_cms914c44d277b1
+LocalJobId                       34033351
+LocalUserId                      cmspilot
+Memory                           20480
+Memory_description               RSS
+Memory_metric                    total
+Memory_storageUnit               kB
+Njobs                            1
+ProbeName                        slurm:cms.rc.ufl.edu
+Processors                       8
+Processors_metric                total
+ProjectName                      avery
+Queue                            hpg1-compute
+RawXML                           (See below)
+RecordId                         cms.ufhpc:3444492.622
+ReportableVOName                 cms
+ResourceType                     Batch
+SiteName                         UFlorida-HPC
+StartTime                        March 26th 2019, 19:17:32.000
+Status                           0
+VOName                           /cms/uscms/Role=pilot/Capability=NULL
+WallDuration                     1494
+checksum                         ad6d91a9e96da9c6b444381f6d16f1bb
+lag_time                         89.16
+lag_time_hr                      0.025
+type                             JobUsageRecord
+
+
```

Comment:
Let's remove the GRACC-record stuff since that's more relevant for the GRACC than the gratia-probe repo.